style.css

```diff
@@ -0,0 +1,67 @@
+.date-select {
```

Comment:
Try using css by composition. Like display-flex is used multiple times. So create a class named DisplayFlex and add to all the elements that will need it.